src/ooxml/java/org/apache/poi/xddf/usermodel/chart/XDDFChart.java

```diff
@@ -301,6 +302,22 @@ public XDDFTitle getTitle() {
         }
     }
 
+    /**
+     * Get or Add chart 3D view into chart
+     * 
+     * @return this method will add 3D view
+     */
+    public XDDFView3D getOrAddView3D()
+    {
+        CTView3D view3D;
+        if(chart.isSetView3D()) {
+            view3D = chart.getView3D();
+        }
+        else {
```

Comment:
The project's code style has the two above lines as one only
```
        }  else {
```

Reply:
updated code